ci-create-docker-server.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Based on the script from Milen Pivchev in nextcloud/ios repository
+
+# This script installs docker and starts a nextcloud test instance
+
+echo "Setup docker..."
+
+brew install docker
+colima start
```

Comment:
Is this faster than using the 3rd party github action?

Reply:
Yes, because the 3rd party action always runs a update of homebrew which is not needed here.

Reply:
Hmm, not sure it's a good idea to expect different states in a test. IMO it's better to reset the state and test only one kind of state. In the Files integration and ui tests I reset all accounts before testing log in.

Reply:
I think this was meant for the other conversation, right? :)
In general I totally agree. Each test should be independent, but since the runners on macOS are ridiculously slow, I went this way for now. 
If we find the time, we can rework some stuff in talk-ios and add some Startup parameters which allow us separate these tests.